scala/README.md

```diff
@@ -534,22 +620,73 @@ Tips & Tricks
 
     ```scala
     // Value classes
-    case class Email(email: String) extends AnyVal // no extra object allocation at runtime
+    case class Email(email: String) extends AnyVal // No extra object allocation at runtime
 
-    val email = Email("no@one.com") // never gets mixed with regular strings
+    val email = Email("joe@example.com") // Never gets mixed with other strings
 
     // Extension methods
     implicit class IntOps(val n: Int) extends AnyVal {
       def stars = "*" * n
     }
 
-    5.stars // equivalent to a static method call, no implicit conversion actually takes place
+    5.stars // Equivalent to a static method call, no implicit conversion actually takes place
     ```
 
 0. Leverage parallel collections, use `.par` judiciously.
 
 0. Use the `@tailrec` annotation to ensure the compiler can recognize a recursive method is tail-recursive.
 
+0. You can use the `@scala.beans.BeanProperty` and `@BooleanBeanProperty` annotations to automatically generate JavaBeans style getter and setter methods.
+
+0. Use this syntax to pass a sequence as parameter to a variable length argument list method:
+
+    ```scala
+    def foo(args: Int*) = ???
+
+    val seq = Seq(1, 2, 3)
+
+    foo(seq:_*)
```

Comment:
i've been doing `foo(seq: _*)`... what is actually happening though. it is casting to a type `_*` right? seems like a space after the colon makes sense in that case.